src/views/Project/index.jsx

```diff
@@ -0,0 +1,37 @@
+import { Component } from 'react';
+import Typography from 'material-ui/Typography';
+import ReactMarkdown from 'react-markdown';
+import projects from '../../data/loader';
+import BugsTable from '../../components/BugsTable';
+
+class Project extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      fileName: props.match.params.projectKey,
+      projects,
+    };
+  }
+  render() {
+    const { fileName, projects } = this.state;
+    const projectInfo = projects[fileName];
+
+    return (
+      <div>
+        <header>
+          <Typography variant="display3" align="center">
+            {projectInfo.name}
+          </Typography>
+          {projectInfo.description && (
+            <Typography align="center">
+              <ReactMarkdown source={projectInfo.description} />
+            </Typography>
+          )}
+          <BugsTable />
```

Comment:
The current UI looks like:

<img width="1680" alt="screen shot 2018-05-26 at 11 18 34 am" src="https://user-images.githubusercontent.com/3766511/40577614-92cbf274-60d6-11e8-8d0e-0d7918fa2097.png">

The UI I have in mind:

<img width="1680" alt="screen shot 2018-05-26 at 11 17 18 am" src="https://user-images.githubusercontent.com/3766511/40577607-643522c8-60d6-11e8-99ec-6598d9e40091.png">